4-css-grid/style.css

```
@@ -0,0 +1,41 @@
+body {
+    display: grid;
+    gap: 5px;
+    grid-template-areas: 'boxa boxa boxb'
+                        'boxc boxd boxb';
+}
+#boxa, #boxb, #boxc, #boxd {
```

```suggestion
.box {
```
Use a classname, it makes more sense for what you're trying to achieve: reusing the same properties.